static/templates/single_message.hbs

```diff
@@ -7,8 +7,10 @@
     {{/if}}
     <div class="messagebox {{#if next_is_same_sender}}next_is_same_sender{{/if}}"
       {{#if msg/is_stream}}style="box-shadow: inset 2px 0px 0px 0px {{background_color}}, -1px 0px 0px 0px {{background_color}};"{{/if}}>
-        <div class="messagebox-content">
-            {{> message_body}}
+        <div class="messagebox-content-wrapper">
```

Comment:
Why is a new wrapper needed? `border` and `outline` already apply outside of `padding`. We keep many hundreds of these in the DOM, so it’s important not to bloat them unnecessarily.

Reply:
Oh I just have a fuzzy background in CSS and don't always know the right way to do this kind of thing (my work experience is mostly in the broad middle of the stack), so thanks!

Reply:
Oh right, part of this was that outline/border was being weird because of the way the timestamp is rendered. I'll poke at it a bit more